### ISSUE TITLE
Useless code

### ISSUE THREAD

**Issue by shiyipaisizuo** [tensorlayer/tensorlayer#738]
### New Issue Checklist

- [X] I have read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [X] I searched for [existing GitHub issues](https://github.com/tensorlayer/tensorlayer/issues)

### Issue Description

This line of code is not used in the full article. It is recommended to remove it.

### Reproducible Code

- Which OS are you using ?
 I:Mac os 13.4
- Please provide a reproducible code of your issue. Without any reproducible code, you will probably not receive any help.
I:OK.

[INSERT CODE HERE]

```python
# define cost function and metric.
y = network.outputs
cost = tl.cost.cross_entropy(y, y_, name='cost')
correct_prediction = tf.equal(tf.argmax(y, 1), y_)
acc = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))

#################################################################
# y_op = tf.argmax(tf.nn.softmax(y), 1)
#################################################################
```

**Comment by DEKHTIARJonathan** [comment]
What file do you talk about ? I don't understand

**Comment by shiyipaisizuo** [comment]
I'm terribly sorry.

**https://github.com/tensorlayer/tensorlayer/blob/master/example/tutorial_mnist_simple.py** +35

**Comment by zsdonghao** [comment]
hi, yes, `y_op = tf.argmax(tf.nn.softmax(y), 1)` is useless in this code, but it is useful when users want to know the way to obtain the output. That is why I put it here.

**Comment by shiyipaisizuo** [comment]
Perfect answer, I will close this question tomorrow, thank you! Changyu Liu 邮箱：shiyipaisizuo@gmail.com 签名由 网易邮箱大师 定制 On 07/10/2018 23:09, Hao wrote: hi, yes, y_op = tf.argmax(tf.nn.softmax(y), 1) is useless in this code, but it is useful when users want to know the way to obtain the output. That is why I put it here. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.
